modules/exploits/windows/local/anyconnect_path_traversal_lpe.rb

```diff
@@ -0,0 +1,213 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Windows::FileInfo
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Cisco AnyConnect Priv Esc through Path Traversal',
+        'Description' => %q{
+          The installer component of Cisco AnyConnect Secure Mobility Client for Windows
+          prior to 4.8.02042 is vulnerable to path traversal and allows local attackers
+          to create/overwrite files in arbitrary locations with system level privileges.
+
+          The attack consists in sending a specially crafted RPC request to the TCP port
+          62522 on the loopback device, which is exposed by the Cisco AnyConnect Secure
+          Mobility Agent service. This service will then launch the vulnerable installer
+          component (`vpndownloader`), which copies itself to an arbitrary location
+          before being executed with system privileges. Since `vpndownloader` is also
+          vulnerable to DLL hijacking, a specially crafted DLL (`dbghelp.dll`) is created
+          at the same location `vpndownloader` will be copied to get code execution with
+          system privileges.
+
+          This exploit has been succesfully tested against Cisco AnyConnect Secure
+          Mobility Client versions 4.5.04029, 4.5.05030 and 4.7.04056 on Windows 10
+          version 1909 (x64) and Windows 7 SP1 (x86).
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'Yorick Koster', # original PoC, analysis
+            'Antoine Goichot (ATGO)', # PoC
+            'Christophe De La Fuente' # msf module
+          ],
+        'Platform' => [ 'windows' ],
+        'Arch' => [ ARCH_X86, ARCH_X64 ],
+        'SessionTypes' => [ 'meterpreter' ],
+        'Targets' => [[ 'Auto', {} ]],
+        'Privileged' => true,
+        'References' =>
+          [
+            [ 'Cisco Bug ID', 'CSCvs46327' ],
+            [ 'URL', 'https://ssd-disclosure.com/ssd-advisory-cisco-anyconnect-privilege-elevation-through-path-traversal/'],
+            [ 'URL', 'https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-ac-win-path-traverse-qO4HWBsj'],
+            [ 'CVE', '2020-3153']
```

Comment:
Unbalanced whitespace.

```suggestion
            ['Cisco Bug ID', 'CSCvs46327'],
            ['URL', 'https://ssd-disclosure.com/ssd-advisory-cisco-anyconnect-privilege-elevation-through-path-traversal/'],
            ['URL', 'https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-ac-win-path-traverse-qO4HWBsj'],
            ['CVE', '2020-3153']
```